### ISSUE TITLE
One game out of bundle is not available for installation via app

### ISSUE THREAD

**Issue by nknauth** [itchio/itch#1576]
- itch client version: 23.6.0
- OS: Windows 8.1 pro, 64-bit

Purchased [The 2017 Fantastic Arcade Bundle](https://juegosrancheros.itch.io/fantastic-arcade-2017) and attempted to install Panic Variants via the itch.io app, but the game did not appear in the install list for the bundle. The game correctly appears tagged as available for download on Windows on the website:

![image](https://user-images.githubusercontent.com/226594/33791026-af5cfd1c-dc4b-11e7-998c-5735b6c269ec.png)

The game does not appear in the download list in the app:

![image](https://user-images.githubusercontent.com/226594/33791032-c53e66c0-dc4b-11e7-9978-16e57d1b894f.png)

I've cleared the itch.io app's cache to attempt to eliminate outdated cache info as a problem source.

[Brandon Boyer of Fantastic Arcade](https://twitter.com/brandonnn/status/938967281580871680) and [Amos](https://twitter.com/fasterthanlime/status/939207606316535808) both confirm that the game is correctly tagged as downloadable for windows.

When downloading the game from the website, it does indeed contain a windows executable, along with a number of other files.

**Comment by fasterthanlime** [comment]
So, this is another bug that results from the following combination of factors (/cc @leafo)

  * There's no way in the itch.io backend to tag executables as 32-bit or 64-bit
  * The itch.io app assumes that one game page ~= one install target (which is true for most pages, the Fantastic Arcade Bundle pages are notable exception)
  * Therefore, the app tries to narrow down the list of uploads by doing text matching

In particular for itch v23.x (the currently released version), the logic in [this code](https://github.com/itchio/itch/blob/57b602575eca6c9cabe1b9092027e4f1ef3aa22c/appsrc/reactors/tasks/queue-game.ts#L112-L149) is as follows:

  * If we have more than 1 upload after all the other filters
  * And there's uploads containing both the string `64` and the string `32`
  * And we're on 64-bit windows
  * Then filter out uploads containing the string `32`

The upload for `Panic Variants` is named `fa_triptych-win32-x64-11-16-0606pm.zip`, which contains the string `32`, so it's filtered out. It's unfortunate, but the real fix is to allow tagging `32-bit` or `64-bit` properly in the backend, regardless of the upload's filename (/cc @leafo again).

Note that I foresee manual tagging of uploads by developers to continue being unreliable in the future, for itch v25.x (the upcoming version of the app), I've developed a series of tools to detect upload types automatically (both platforms and architecture), see the [configurator](https://github.com/itchio/butler/tree/master/configurator) package mostly, which leaf & I discussed integrating into our build processing backend (the one used for builds pushed with [butler](https://itch.io/docs/butler/)), it just hasn't been done yet.

---

A quick workaround would be to change the upload record to have a filename like `fa_triptych-win-x64-11-16-0606pm.zip` (removing `32` from the string) - which is dirty and bad, but can be done in a minute rather than waiting for the root of the problem to be addressed.
